### PR TITLE
Arconomy: The big balance PR.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

This PR does a bunch of shit that I kinda wish I held off on doing right away but someone said "Do it no balls" and you now you know why you're here.

Alright so this hits some of the major pressure points on economy to help it regain a semblance of balance on live. Most noticeably this will cover 3 major areas.
* **Price Shifting.**
* **Passive Income.**
* **Gas Exports.**

First let me go and link everyone to my relevant Design Document:
https://hackmd.io/WlWgyRafTaiAqz6ouOqC-Q
Cool, welcome back. I assume you read all of it? Sorry for the long read but this was rather spur of the moment.

### Price Shifting
So, in-game items that have prices have a major issue on their hands, being that they were decided in regards, roughly to how much money that job should make. This means that many of the jobs in-game have been given prices scaled to their job's income. That income I adjusted by removing passive income in #54161. While this was helpful to moving towards an active in-game economy, it resulted in items falling into distinct price brackets. A high paying job like security's items could never be purchased by someone like a botanist, but a job like a security officer had more capital and buying power than most other jobs in-game combined when moving down those brackets. We've done a simple replacement of scale to help in bring things closer to a semblance of equality.

There are now 3 price brackets, `PAYCHECK_LOW`, `PAYCHECK_CREW`, and `PAYCHECK_COMMAND`. Command staff will still have a higher base level of money on-hand than other crew, and low paying wages that we on-station don't respect as being real jobs (assistant, prisoner) will have their items be intentionally cheaper to encourage active participation, but the difference in scale is now noticeably far closer to each other.

What this means is that in-game players will have more options in regards to what a single job will allow them to interact with the economy. It also means we arbitrarily enforce a system that allows for items to have uniformity in what they cost to other players. 50 credits for a wrench feels better when you know that other job critical items in-game are also around the same price, and it's equivalent to one paycheck.

### Wait did you just say paycheck
Okay so bare with me on this:
Economy lost it's relationship to time. In a game where a single round takes 90+ minutes, having a relationship to time and how long it takes to afford something is a major consideration when you look at something. Also, we get to say that I was certifiably wrong in regards to the active economy thing, since we have very, VERY few active sources of content in-game that are very... fun? Bounties are literal fetch quests but something like tourists is at least more engaging and interactive with the round.

So, alongside the fact that comparatively player's wallets and base income have been lowered to scale, they'll get paychecks back. In-game price inflation will mean that prices will still fluctuate, and go up if players as a whole don't spend money. 
I'm very willing to hear feedback on this here and from maintainers.

As an additional note, doing this meant tweaking down the syndicate briefcase of cash, so that instead of giving you 5000 credits, which is now EXTREMELY HIGH, it instead gives the player 500 credits, effectively doubling the player's starting capital.

### Gas Exports.
ALRIGHT ARE YOU READY FOR SOME GRAPHS I THOUGHT SO YOU LOVE GRAPHS.
So, gas exports are fucked, have always been fucked, and consistently have proven to be capable of breaking the in-game economy for a long time. This is no secret, I've been pinged with players getting billions, actual billions of credits using it multiple times in as many years. See, any round where a player manages to buy the bicycle is a round where I've fucked up, or someone fucked and I let it get past me.

So here's how gas exports work right now.
image
So the vertical axis is the value of gas in a can, and the horizontal axis is the number of moles of a gas in that can.
So, all of this hinges on the value of a single mole of gas, and some gasses enable you to make extremely, EXTREMELY profitable gasses through atmospheric gas wizardry. As you can see by the green line. However, even those less profitable gasses are still in an extremely high magnitude of value.
image
Most gasses if you have a full can of it will net you OVER 10k credits. For scale, one crate being sold in cargo is 200 credits.
That's a minimum of crates for pumping gas into a hollow metal box and praying it doesn't explode.

So we adjusted the values accordingly.
DesmosAtmosGasCaps
The baseline value of a single gas has been tweaked downward signifigantly. Even these values are still arguably very high, but I can play with it at the discretion of LemonintheDark. The green line at the top represents gasses that previously sold for 100 credits per mole, antinobilium I believe, and working downwards. I am going to try and enforce 10 credits per mole as the absolute maximum hard cap on gas exports, regardless of how many gasses we try to add in the future. Because the alternative is getting a gunjillion credits by huffing miasma into a tank of steel. And we ain't having that shit.

## Why It's Good For The Game

For many of the reasons that I outlined above, this is a good change in a positive direction.
Players get more ability to interact with the economy without having to do content that's becoming increasingly depreciated in my absence.
Players also have a baseline consensus on what values of credits are high and low because jobs have been given an equalized standard in regards to the cost of certain items.
Price fluctuations through inflation will now be more meaningful in situations where the economy becomes more relevant.
The system will still encourage you to play a job that's productive to the status of the station through lower paycheck jobs existing as well.
Gas exports are now reduced to the point that their value is appropriate for the first time... actually ever. Nice.

## Changelog

:cl:
balance: The values of nearly every item purchasable by players has been rebalanced.
balance: Players will now start with less starting money, but will receive a paycheck once every 5 minutes.
balance: The value of gasses exported through the cargo department have been skewed way, WAY down in terms of price.
balance: The Syndicate briefcase of cash now contains an appropriate 500 credits instead of 5000 credits.
/:cl:

### CODE REVIEW

**Comment by GreatKingWombat** [comment]
RIP briefcase of cash. It was fun while it lasted

**Comment by Pickle-Coding** [comment]
I'd nerf nitrium further, and buff hypernoblium. Nitrium is cheap and easy to make, and hypernoblium is very expensive to produce and should be in the mid tier ranges.

**Comment by Cheshify** [comment]
A few things:
Atmos' biggest issue with gas prices is that there's no relation between Difficulty of Gas Production and Price of Gas. Nitrium isn't really that hard for seasoned atmos players to make shitloads of cash off of it because it's 100 credits per mol, while much rarer gasses like Zauker and Healium are worth less than a quarter of that.

Secondly, the change to bring back paychecks is fantastic. Trying to run any assistant shop never ends up working because once people buy their roundstart drip from vendors, they're likely not to have any cash leftover to spend on anything expensive that you could make and sell.

**Comment by GuillaumePrata** [comment]
I just wish Atmos for profit was completely removed, people will stare at the code until they figure out what steps are needed so they can min max their setup again in a month.
Atmos Techs ignoring most of the station burning because they are factory printing money was always silly.

**Comment by Cheshify** [comment]
> I just wish Atmos for profit was completely removed, people will stare at the code until they figure out what steps are needed so they can min max their setup again in a month. Atmos Techs ignoring most of the station burning because they are factory printing money was always silly.

I think that's another issue of atmos having no late-game goals aside from bragging that they made really hard gasses.

**Comment by GuillaumePrata** [comment]
> I think that's another issue of atmos having no late-game goals aside from bragging that they made really hard gasses.

I would say the end game goal should be to keep the station in perfect shape against all the antagonists trying to nuke the place but I know it is not what most players want as gratification.

Atmos shouldn't be Xenobiology where people sit in a room and chase a rainbow slime. (not like this is hard anymore)
Atmos for profit rewards those players with a Bike or guns.

**Comment by RaveRadbury** [comment]
Hoping that we're bullying income for non-humans bit knowing that likely won't be the case. 😔

**Comment by YakumoChen** [comment]
It's hard to glean what exactly this means for bounties, and especially for ordering crates from cargo on your own dime.
The cash briefcases were largely used to self-purchase cargo crates, something doing bounties often allowed you to do as well, and most of the goodies are balanced around doing bounties to afford them as opposed to just buying them off paychecks.

**Comment by YakumoChen** [comment]
(500 credits as an aside doesn't feel like good bribe money if you're actually roleplaying with it)

**Comment by RandomGamer123** [comment]
> I'd nerf nitrium further, and buff hypernoblium. Nitrium is cheap and easy to make, and hypernoblium is very expensive to produce and should be in the mid tier ranges.
> 
> More importantly I think the diminishing returns should be based on per shuttle instead of per canister.



> A few things: Atmos' biggest issue with gas prices is that there's no relation between Difficulty of Gas Production and Price of Gas. Nitrium isn't really that hard for seasoned atmos players to make shitloads of cash off of it because it's 100 credits per mol, while much rarer gasses like Zauker and Healium are worth less than a quarter of that.
> 
> Secondly, the change to bring back paychecks is fantastic. Trying to run any assistant shop never ends up working because once people buy their roundstart drip from vendors, they're likely not to have any cash leftover to spend on anything expensive that you could make and sell.

Highly agree to both of these. Nitrium retained its high price from stimulum in the gas merging update, but was made many times cheaper to produce (although even stimulum was not that hard via fusion).
Antinoblium had a base price 10 times lower than nitrium despite being significantly more difficult to obtain in the quantities that nitrium is, and hypernoblium is the same price as pluoxium, despite the latter being easily obtained through the SM and the former requiring mid-tier tritium-based synthesis or fusion.
Also the "per shuttle" vs "per canister" change, considering how people just split up gas into canisters to sell it, I would say that's a fairer way to balance it than just nerfing all gas selling.

**Comment by bluetshirtguy** [comment]
> Hoping that we're bullying income for non-humans but knowing that likely won't be the case. 😔

The amount of people that have fun being stepped on is staggeringly low.

**Comment by Sealed101** [comment]
> (500 credits as an aside doesn't feel like good bribe money if you're actually roleplaying with it)

i once bribed a catgirl into jumping off the eshuttle mid transit with 4 briefcases of cash. i highly doubt anyone would pull off the same feat for 2 grand instead of 20 grand so i support the concern of syndicate bribery budget cuts

**Comment by DrDiasyl** [comment]
Great PR, I agree that economy is kinda shitty and useless right now.

**Comment by Stewydeadmike** [comment]
I'll also echo the sentiment that 500 credits seems kind of weak as a non-violent bargaining chip, but I suppose it's hard to really tell without seeing it in action alongside the rest of the changes.

**Comment by anclandor** [comment]
The 500 credits is pretty lame.  The point of the briefcase isn't to have slightly more money than the average spaceman, its to be able to buy whatever you want and still have tipping money left over or to bribe people with insane amounts of cash.

**Comment by moocowswag** [comment]
500 Credits isnt even worth having a briefcase for, it should be summoned in a wallet.

**Comment by BrandonSmith6** [comment]
I think atmos export prices should be changed but significantly harder gases should have a significantly higher price. It just looks like a nerf to atmos export entirely even the hardest gases out there are being nerfed into the ground.

File: code/controllers/subsystem/economy.dm [hunk 1]
```diff
@@ -83,6 +83,7 @@ SUBSYSTEM_DEF(economy)
 		var/datum/bank_account/bank_account = bank_accounts_by_id[account]
 		if(bank_account?.account_job && !ispath(bank_account.account_job))
 			temporary_total += (bank_account.account_job.paycheck * STARTING_PAYCHECKS)
+		bank_account.payday(1, FALSE)
```
Comment (LemonInTheDark) [inline code/controllers/subsystem/economy.dm:86]: Why the false here? isn't it implicit?

**Comment by LemonInTheDark** [comment]
It is a nerf to atmos export prices. I told arcane he should probably just nuke them, but he's decided to try and balance them instead. He's trying to keep the feature despite the trouble it's caused in the past, I think that's respectable.

As for the rest of this pr.
I like a more defined set of price brackets.
It's sad to see active income fall to the wayside, I liked the concept of it quite a lot, a reason to actually sell things is really nice. I hope paychecks are low enough that it's still worth doing.

I'll review the price values later, but I in general trust you to at least move us in a better direction then before.

> Hoping that we're bullying income for non-humans but knowing that likely won't be the case. 😔

Do non humans get paid less currently? If so, yes

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

**Comment by LemonInTheDark** [comment]
> I'll also echo the sentiment that 500 credits seems kind of weak as a non-violent bargaining chip, but I suppose it's hard to really tell without seeing it in action alongside the rest of the changes.

Thinking about it in terms of time, it's 10 paydays for normal crew, 5 for command, and 20 for assistants.
That works out to about 50, 25 and 100 minutes worth of time.

Will need to read the rest of the changed numbers to see if that makes sense, Seems like a good amount to buy someone over to you, but maybe it could use tweaking

**Comment by moocowswag** [comment]
> > I'll also echo the sentiment that 500 credits seems kind of weak as a non-violent bargaining chip, but I suppose it's hard to really tell without seeing it in action alongside the rest of the changes.
> 
> Thinking about it in terms of time, it's 10 paydays for normal crew, 5 for command, and 20 for assistants. That works out to about 50, 25 and 100 minutes worth of time.

Without taking passive income, you can easily make 500 credits by doing a couple of bounties. Anyone that actually wants to pursue any amount of cash would easily make 500 bucks pretty quickly.

**Comment by ArcaneMusic** [comment]
I am more than happy to remove cases of cash.

Cases of cash are difficult to model in regards to the health of economy on a given round, being that:
Players can purchase a large quantity of them in a given round, as they're 1TC. 1TC traitor tasks are easy to come by, and buying a single one of these, as has been previously stated, advances a player's economic status upwards of 50 minutes forward through purely passive income. I'm leaving them in at a reduced level, and if they need to be buffed, or nerfed, going forward, that's fine.

Cases of cash were first introduced into the game in #6871. In 2015, we didn't have an economy, money trees were a funny meme that nobody actually grew, and cargo points were still getting the work around. When they were first introduced, they were added as 5000 credits, and have unchanged ever since. I think it's pretty clear that they weren't added with any clear intent in mind.
image
It's perfectly acceptable to balance something that would otherwise create unhealthy gameplay.

**Comment by LemonInTheDark** [comment]
It may be worth raising the tc cost. So the case is still a useful motivator, but not quite as bottom of the barrel cheap. S not like it was balanced before anyway, what's important would be hitting the sweetspot of enough money to convince people to do stuff for you. Making the economy more standardized will help with that a good deal

**Comment by YakumoChen** [comment]
> #6871

Jesus christ, it's bad that I know most of the people in this PR, and some of them still even play this godforsaken game.

**Comment by LemonInTheDark** [comment]
Was this closed for lack of interest?

**Comment by Pickle-Coding** [comment]
Oranges said they were closed for being stale.

**Comment by LemonInTheDark** [comment]
@ArcaneMusic you alive?

**Comment by ArcaneMusic** [comment]
I am alive but I only just became alive this week due to academic deadlines. If the one outstanding review is fixing the one var on payday then I don't see any reason it couldn't go for a TM for a round.

**Comment by LemonInTheDark** [comment]
Is it something you've got the time to work on? I know you're busy with newscasters too, don't wanna overload ya. Can always wait a bit

**Comment by ArcaneMusic** [comment]
I'm going to give it a few days to try and implement some more high level changes in addition to what I'm already doing and if my availability seems more open I'll see what I can do to get this ready for merge/review this week. I'd like to get this out after newscasters but newscasters will be stuck in review hell for a few centuries still so we'll see 🤷

**Pull request closed without merging**